kornia/augmentation/container/patch.py

```diff
@@ -0,0 +1,267 @@
+from typing import Dict, List, Optional, Tuple, Union
+
+import torch
+import torch.nn as nn
+
+from kornia.augmentation.augmentation import ColorJitter
+from kornia.augmentation.base import _AugmentationBase, IntensityAugmentationBase2D
+from kornia.contrib.extract_patches import extract_tensor_patches
+
+from .image import ImageSequential
+
+
+class PatchSequential(ImageSequential):
+    r"""Container for performing patch-level image processing.
+
+    PatchSequential breaks input images into patches by a given grid size, which will be resembled back
+    afterwards. Different image processing and augmentation methods will be performed on each patch region.
+
+    Args:
+        *args (nn.Module): a list of processing modules.
+        same_on_batch (bool, optional): apply the same transformation across the batch.
+            If None, it will not overwrite the function-wise settings. Default: None.
+        keepdim (bool, optional): whether to keep the output shape the same as input (True) or broadcast it
+            to the batch form (False). If None, it will not overwrite the function-wise settings. Default: None.
+        patchwise_apply (bool, optional): apply image processing args will be applied patch-wisely.
```

Comment:
```suggestion
        same_on_patch (bool, optional): apply image processing args will be applied patch-wisely.
```

Reply:
for consistency

Reply:
Nono, it is a different concept. Check out https://app.reviewnb.com/kornia/tutorials/pull/12/.

Reply:
gotcha